web_src/js/markup/math.ts

```diff
@@ -11,9 +11,9 @@ function targetElement(el: Element): {target: Element, displayAsBlock: boolean}
   };
 }
 
-export async function renderMath(): Promise<void> {
-  const els = document.querySelectorAll('.markup code.language-math');
-  if (!els.length) return;
+export async function initMarkupCodeMath(elMarkup: HTMLElement): Promise<void> {
+  const el = elMarkup.querySelector('code.language-math'); // .markup code.language-math'
+  if (!el) return;
```

Comment:
Regression, here and other similar places: it only initializes the first element, others are not initialized.

The fix is done in  Fix markdown render behaviors #34122